mapdata.rb

```diff
@@ -1,16 +1,15 @@
 require "csv"
 ary = CSV.read("header.csv")
-#array = CSV.read("exampledata.csv")
+array = CSV.read("exampledata.csv")
 library = []
 #CSV.open("exampledata.csv", "wb") do |row|
 #data_array = array.map do |value|
-  ary.each do |key|
-    data_array = array.map do |value|
-  #  puts value
-  puts key[0]
-
-  x = {}
-  #x.store(key.to_s, value)
-  #puts x.inspect
-  end
+array.each do |value|
+  #ary.each do |key|
+  #  puts value[0].inspect
+    #  puts value
+    x = {}
+    x.store(:hello, value[0])
+    puts x.inspect
```

Comment:
`x.inspect` is just showing you everything. It's usually used only for logging / debugging purposes. To just print a value, you do this.

```rb
puts x[:hello]
```